arm.scad

```diff
@@ -12,26 +13,42 @@ grip_width = 2;
 extend_retract_gear_size = 1;
 
 // Begin how to print this piece (orientation, etc)
-rotate([0, 90, 0]) arm();
+rotate(270, y) arm();
+claw();
 
 // Begin modules that can be used to assemble this piece in solver_3.scad
 module arm(include_gears=true) {
-  rotate([0, 270, 0]) {
-    translate([0, 0, cube_size/2])
-      if (include_gears)
-        opposing_gears(arm_length, width, extend_retract_gear_size);
-      else cylinder(d=width, arm_length);
-    difference() {
-      translate([0, 0, cube_size/3/2 + grip_width / 2])
-        ccube([
-          cube_size + grip_width * 2,
-          cube_size / 3,
-          cube_size/3 * 2 + grip_width
-        ], x + y + z);
-      ccube(cube_size, x + y + z);
+  difference() {
+    union() {
+      rotate([0, 90, 0]) {
+        if (include_gears)
+          opposing_gears(arm_length, width, extend_retract_gear_size);
+        else cylinder(d=width, arm_length);
+      }
+      ccube([grip_width*4.5, grip_width*8, grip_width*2 + cube_size/3], y + z);
     }
+    up(-cube_size/3/2)
+      for (i=[0:1])
```

Comment:
intsead of `for() rotate(i*...) translate(i*...) shape()`, you can do `mirrored([x]) shape()`